dump26/‏‏claims_new/db_log.py

```
@@ -0,0 +1,255 @@
+from pathlib import Path
+
+main_path = Path(__file__).parent
+
+db_path = f"{str(main_path)}/wd_logs.db"
+
+db_path_main = {1: str(db_path)}
+
+print("db_path", db_path_main[1])
+
+
+def db_commit(query, params=[], many=False):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)